### ISSUE TITLE
integration between google docs and ansible?

### ISSUE THREAD

**Issue by davidelang** [socallinuxexpo/scale-network#249]
We have a number of source-of-truth docs that are current .tsv files

google docs has versioning, branching, and user permissions, and in addition will allow simultaneous edits (avoiding merging issues)

Can we find some way to pull the source-of-truth files from google docs (ideally spreadsheets) and use them locally? In git this may be a trigger that happens on a pull to get the docs, or we may want to pull them into git and version them daily (in case someone doesn't tag a version)

**Comment by kylerisse** [comment]
related to #232

**Comment by kylerisse** [comment]
Doing this as a 2 part process. The first portion is to have the Ansible inventory script consume a CSV  instead of a TSV which is included in PR #252 . The second portion will be a future effort to implement a new sync script from Google Sheets to populate the CSV that Ansible consumes.

**Comment by owendelong** [comment]
I'm not sure I'm wild about converting everything to csv vs. tsv. Having said that, if we're really determined to move forward with it, then I think the Google sheet integration is a critical first step rather than the other way around.

Editing CSVs is a huge PITA and for the most part, I've been the one maintaining these files and using vi to do it.

If we're dead set on doing this, then there's a python script as part of this project (https://www.sparkfun.com/sparkx/blog/2591)  which extracts a CSV from a google sheet automatically that you might find useful.

**Comment by kylerisse** [comment]
@owendelong this has nothing to do with the files you maintain and they have been omitted precisely because of this objection. We discussed this at SCaLE per @davidelang's request. It is captured in #232. This only applies to APs, Pis, servers, and routers which are all files you don't touch. Your switch configuration scheme remains unchanged and the reason I requested @davidelang and @sarcasticadmin review as they are the ones impacted.
